net/socket_vmnet/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        lima-vm socket_vmnet 1.1.3 v
+
+categories          net
+maintainers         {sub-pop.net:link @subpop} \
+                    openmaintainer
+license             APL-2.0
+
+description         vmnet.framework support for rootless QEMU
+long_description    Daemon to provide access to vmnet.framework for rootless QEMU
+homepage            https://github.com/lima-vm/socket_vmnet
+
+checksums           rmd160 c72c72fe8471df6d31df287d8df843e49d3a1fed \
+                    sha256 f5ec3fa24ad2c857b2b555f4fc91b39cbf3e9c4bc3191fb64c46ee7f49027b47 \
+                    size 18738
+
+use_configure       no
+build.args          VERSION=${version} \
+                    SOURCE_DATE_EPOCH=1696806366 \
+                    PREFIX=${prefix}
+destroot.target     install.bin install.doc
+destroot.args       ${build.args}
+
+startupitem.create     yes
+startupitem.logfile    /var/log/${name}.log
+startupitem.user       root
+startupitem.executable ${prefix}/bin/socket_vmnet --vmnet-gateway=192.168.105.1 /var/run/socket_vmnet
```

Comment:
Given that this sets up a startup item, you might want to alert the user using the "notes" facility. Other than that, this LGTM.

Reply:
Oh, thanks for pointing that out. This feels like it should be a fairly standard
message to report to users; is there any example I can follow?

Reply:
I also noticed that during installation, `port` prints a message about the startupitem and how to load it. Do you think an additional message using `notes` is really necessary too?

```
--->  Fetching archive for socket_vmnet
--->  Attempting to fetch socket_vmnet-1.1.3_0.darwin_23.arm64.tbz2 from https://packages.macports.org/socket_vmnet
--->  Attempting to fetch socket_vmnet-1.1.3_0.darwin_23.arm64.tbz2 from http://mirror.fcix.net/macports/packages/socket_vmnet
--->  Attempting to fetch socket_vmnet-1.1.3_0.darwin_23.arm64.tbz2 from https://mse.uk.packages.macports.org/socket_vmnet
--->  Fetching distfiles for socket_vmnet
--->  Attempting to fetch socket_vmnet-1.1.3.tar.gz from https://distfiles.macports.org/socket_vmnet
--->  Attempting to fetch socket_vmnet-1.1.3.tar.gz from https://codeload.github.com/lima-vm/socket_vmnet/legacy.tar.gz/v1.1.3?dummy=
--->  Verifying checksums for socket_vmnet
--->  Extracting socket_vmnet
--->  Configuring socket_vmnet
--->  Building socket_vmnet
--->  Staging socket_vmnet into destroot
--->  Creating launchd control script 'socket_vmnet'
--->  Installing socket_vmnet @1.1.3_0
--->  Activating socket_vmnet @1.1.3_0
--->  Cleaning socket_vmnet
--->  Updating database of binaries
--->  Scanning binaries for linking errors
--->  No broken files found.
--->  No broken ports found.
--->  Some of the ports you installed have notes:
  socket_vmnet has the following notes:
    A startup item has been generated that will aid in starting socket_vmnet
    with launchd. It is disabled by default. Execute the following command to
    start it, and to cause it to launch at startup:

        sudo port load socket_vmnet
```

Reply:
That might be enough. You can grep for other Portfiles that activate startup items and look at their notes too, if you judge that the existing note is sufficient, you can tell me and I'll just commit it as is.

Reply:
I did look at some other ports. It seems many of their notes were related to other content or only slightly related to the startup items. I feel like this auto-generated note is better than anything I could write manually, so I think this Portfile is ready to commit.

Reply:
Done.